app/src/main/java/com/duckduckgo/app/notification/NotificationHandlerService.kt

```diff
@@ -109,6 +132,8 @@ class NotificationHandlerService : IntentService("NotificationHandlerService") {
         const val APP_LAUNCH = "com.duckduckgo.notification.launch.app"
         const val CLEAR_DATA_LAUNCH = "com.duckduckgo.notification.launch.clearData"
         const val CANCEL = "com.duckduckgo.notification.cancel"
+        const val WEBSITE = "com.duckduckgo.notification.website"
+        const val CUSTOMIZE_ICON_FEATURE = "com.duckduckgo.notification.app.feature.customizeIcon"
```

Comment:
can you change this to `CHANGE_ICON_FEATURE`? just to be consistent with the `Activity` and messaging around the feature